mteb/tasks/Retrieval/slk/SlovakSumRetrieval.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class SlovakSumRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="SlovakSumRetrieval",
+        description="""
+            SlovakSum, a Slovak news summarization dataset consisting of over 200 thousand
+            news articles with titles and short abstracts obtained from multiple Slovak newspapers.
+
+            Originally intended as a summarization task, but since no human annotations were provided
+            here reformulated to a retrieval task.
+        """,
+        reference="https://huggingface.co/datasets/kiviki/SlovakSum",
+        dataset={
+            "path": "kiviki/SlovakSum",
+            "revision": "85d6b32f2762313714618171b9d1a65eb7408835",
+        },
+        type="Retrieval",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["slk-Latn"],
+        main_score="ndcg_at_1",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license="openrail",
+        socioeconomic_status=None,
+        annotations_creators="derived",
+        dialect=None,
+        text_creation=None,
```

Comment:
Please add thse

Reply:
Done. The article doesn't mention the dates when the data was scraped from but I asked the author, so will that fill in when (if) they send me that info.